### DISCUSSION TITLE
Enhancement: Dashboard Widget - Updates

### DISCUSSION THREAD

**Discussion by richhayler** [craftcms/cms#8674] [Ideas]
When you have one or more expired plugins with updates the "Updates" widget shows you have updates when you can't actually update the plugins due to the subscription expiration.

This is annoying and reduces the usefulness of the updates widget because in this situation the "check for updates" button disapears so you don't have the facility to click the button and check if you have updates for plugins with an active subscription (which can be updated).

In my scenario I have one plugin I have purposely chosen not to renew the subscription for now and the updates widget is showing I have one update (which is for this expired plugin). This is persistent so for a couple of days I've been thinking I only have one update available.

Coincidentally today I clicked on the updates widget link “show updates” and there are in fact now other plugin updates showing.

This is frustrating as I believe if you have expired plugins then those notifications should be separated from updates notifications.

Maybe it would be better for the updates widget to look something like the following.

- One update available! Go to Updates _(for plugins with active subscriptions)_
- Two plugins have expired! Review Renewals

and

- Congratulations! You’re up-to-date.
- Two plugins have expired! Review Renewals
[Check for updates]

**Comment by brandonkelly** [comment]
I think the mute idea suggested in https://github.com/craftcms/cms/issues/4186#issuecomment-489431311 would solve this nicely.
